R/R/executeInSQL.R

```diff
@@ -162,37 +179,61 @@ executeSQLQuery <- function(connectionString, sqlQuery, getScript = FALSE)
               "
     spees <- speesBuilder(script = script, inputDataQuery = sqlQuery, TRUE)
 
-
-    if(getScript) {
+    if(getScript)
+    {
         return(spees)
-    } else {
+    }
+    else
+    {
         execute(connectionString, spees)$result
     }
 }
 
+
 #
-#Execute and process a script
+# Use odbc and connection string to connect to a server
+# @param connectionString character string. The connection String to the database

```

Comment:
another super nit: remove space between connection and String in the description similar to other descriptions.